tests/integration/cli.py

```diff
@@ -31,3 +33,12 @@ def it_shows_help_with_no_arguments(self):
         p = Popen(('pgctl-2015',))
         assert p.wait() == 2  # too few arguments
         # TODO assert the output
+
+
+class DescribeSvscan(object):
+
+    def it_does_not_deadlock(self, in_example_dir):
+        idempotent_svscan('bad_service')
```

Comment:
Not in love with this test because failure of the test hangs the system, I could do this with a sleep instead... not sure if that is much bettter